server_admin/topics/roles/composite.adoc

```diff
@@ -2,20 +2,21 @@
 
 === Composite Roles
 
-Any realm or client level role can be turned into a _composite role_.
-A _composite role_ is a role that has one or more additional roles associated with it.
-When a composite role is mapped to the user, the user also gains the roles associated with that composite.  This inheritance
-is recursive so any composite of composites also gets inherited.
+Any realm or client level role can be a _composite role_. A _composite role_ is a role that has one or more additional roles associated with it. When a composite role is mapped to a user, the user also gains the roles associated with that composite.  This inheritance is recursive so any composite of composites also gets inherited.
```

Comment:
After this paragraph, add "However, we recommend that composite roles are not overused."